libraries/AP_Scripting/applets/switch_rates.lua

```diff
@@ -0,0 +1,176 @@
+--[[
+   rate switch utility. This helps to compare different tunes in-flight to compare performance
+--]]
+
+---@diagnostic disable: param-type-mismatch
```

Comment:
This is cheating. What is the error?

Reply:
I think I just copied this from somewhere else!